src/components/Announcements/helpers.ts

```diff
@@ -0,0 +1,13 @@
+import { Announcement } from './types'
+
+export const filterData = (
```

Comment:
nit: Should we name this one with something like `filterSeen` or `getSeenAnnouncements`? `filterData` seems a bit too generic to easily understand it on the caller side

Reply:
Yes you are right. I renamed it `getUnseenAnnouncements` it would be more appropriate